internal/resources/metal/organization/resource_test.go

```diff
@@ -1,62 +1,28 @@
-package equinix
+package organization_test
 
 import (
 	"fmt"
-	"log"
 	"testing"
 	"time"
 
+	"github.com/equinix/terraform-provider-equinix/internal/acceptance"
 	"github.com/equinix/terraform-provider-equinix/internal/config"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
 	"github.com/hashicorp/terraform-plugin-testing/terraform"
 	"github.com/packethost/packngo"
 )
 
-func init() {
-	resource.AddTestSweepers("equinix_metal_organization", &resource.Sweeper{
-		Name:         "equinix_metal_organization",
-		Dependencies: []string{"equinix_metal_project"},
-		F:            testSweepOrganizations,
-	})
-}
-
-func testSweepOrganizations(region string) error {
```

Comment:
The sweeper should not be deleted.  If #593 is merged before this one the sweeper should be updated to match the changes made there.  In the meantime the sweeper should stay as-is.